sc/shared/src/test/scala/sigmastate/utxo/BasicOpsSpecification.scala

```diff
@@ -1734,4 +1804,26 @@ class BasicOpsSpecification extends CompilerTestingCommons
     }
   }
 
+  property("Unit register") {
+    // TODO frontend: implement missing Unit support in compiler
+    //  https://github.com/ScorexFoundation/sigmastate-interpreter/issues/820
+    test("R1", env, ext,
+      script = "", /* means cannot be compiled
+                     the corresponding script is { SELF.R4[Unit].isDefined } */
+      ExtractRegisterAs[SUnit.type](Self, reg1)(SUnit).isDefined.toSigmaProp,
+      additionalRegistersOpt = Some(Map(
+        reg1 -> UnitConstant.instance
+      ))
+    )
+
+    test("R2", env, ext,
+      script = "", /* means cannot be compiled
+                   the corresponding script is "{ SELF.R4[Unit].get == () }" */
+      EQ(ExtractRegisterAs[SUnit.type](Self, reg1)(SUnit).get, UnitConstant.instance).toSigmaProp,
+      additionalRegistersOpt = Some(Map(
+        reg1 -> UnitConstant.instance
+      ))
+    )
+  }
```

Comment:
What this test means?

Reply:
how is this test related to the PR ? Also, why to ask others about tests introduced by you (starting from  [4f2579bbaeec )3def75d42629d7821073f39804b2](https://github.com/ergoplatform/sigmastate-interpreter/commit/4f2579bbaeec3def75d42629d7821073f39804b2)  )